grade.go

```diff
@@ -392,39 +404,27 @@ func main() {
 
 		result.StudentId = hydratedStudent.Id
 		result.AssignmentId = int8(1)
-
-		if language == "python3" || language == "python" {
+		
+		if language == "python3" || language == "python" || language == "javascript" || language == "js" {
 			result.CompileSuccess = true
-
-			stdout := runInterpreted(filepath.Join(workDir, dir), runArgs, input)
+	
+			stdout := runInterpreted(filepath.Join(workDir, dir), runArgs, language, input)
 			fmt.Printf("Output for %s: %s", result.Student, stdout)
 			result.RunCorrect, result.Feedback = compare(expected, stdout)
+			
+		// if student doesn't exist, commit to db
```

Comment:
please check the rebase for other inconsistencies

Reply:
Definitely going to check the rebase now.